depends/common/gnutls/03-patch-connectx-osx.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur gnutls/lib/system/fastopen.c gnutls-patch/lib/system/fastopen.c
+--- gnutls/lib/system/fastopen.c	2020-01-27 20:24:23.000000000 +0000
++++ gnutls-patch/lib/system/fastopen.c	2020-01-27 21:59:09.000000000 +0000
+@@ -37,7 +37,7 @@
+ #endif
+ 
+ /* TCP Fast Open on OSX behaves differently from Linux, so define these helpers */
+-#if defined __APPLE__ && defined __MACH__ && defined CONNECT_DATA_IDEMPOTENT && defined CONNECT_RESUME_ON_READ_WRITE
++#if defined __APPLE__ && defined __MACH__ && defined CONNECT_DATA_IDEMPOTENT && defined CONNECT_RESUME_ON_READ_WRITE && defined HAVE_DARWIN_CONNECTX
```

Comment:
I created this patch @AlwinEsch, but instead of checking for `defined HAVE_DARWIN_CONNECTX` which can never be true I really want to check if the `macosx-version-min` is 10.11 or higher.

Reply:
Ok, instead I just changed the define name so it would skip calling connectx.

Unless we change kodi to min osx version of 10.11 (currently `platform_min_version="macosx-version-min=10.9"`) then this patch is required.